<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Название</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Сорт:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="18" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Степень прожарки:</string>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="24" column="0" colspan="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Молотый/зерновой</string>
     </property>
    </widget>
   </item>
   <item row="28" column="1">
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="30" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Описание:</string>
     </property>
    </widget>
   </item>
   <item row="35" column="1">
    <widget class="QLineEdit" name="lineEdit_6">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="36" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Цена, руб:</string>
     </property>
    </widget>
   </item>
   <item row="39" column="1">
    <widget class="QLineEdit" name="lineEdit_7">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="42" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Объем:</string>
     </property>
    </widget>
   </item>
   <item row="44" column="1">
    <widget class="QLineEdit" name="lineEdit_8">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
